internal/payloads/error_payload.go

```diff
@@ -10,6 +10,23 @@ import (
 	"github.com/go-chi/render"
 )
 
+type APIError interface {
+	Error() string
+	GetStatus() int
```

Comment:
The recommended way for getter is just `Status()` but it's okay. We already have few in the codebase.